src/config/api/index.js

```diff
@@ -61,6 +65,18 @@ api.addRequestTransform(request => {
     const snakeCaseSerialize = new SnakecaseSerializer();
     request.data = snakeCaseSerialize.serialize(request.data);
   }
+
+  const token = LocalStorageService.getValue(TOKEN_KEY);
+  const client = LocalStorageService.getValue(CLIENT_KEY);
+  const uid = LocalStorageService.getValue(UID_KEY);
+
+  if (token) {
+    request.headers = {
+      [TOKEN_KEY]: token,
+      client,
+      uid
+    };
+  }
```

Comment:
i would do this here, because on EVERY request you are querying local storage to get token and set headers, its not optimal